### DISCUSSION TITLE
error loading file: error decoding response body: expected value at line 1 column 1

### DISCUSSION THREAD

**Discussion by deedot** [Marekkon5/onetagger#337] [Q&A] [answered]
Can someone please tell me why the majority of my files are being skipped with the message above?

Thank you so much in advance!!

**Comment by Marekkon5** [comment, marked as answer]
Hello, can you try the latest development build from the Actions tab, it has potential fixes.

**Reply by deedot** [reply]
ok I will try now

**Comment by deedot** [comment]
Im going to be honest, I cant find the development build that you speak of in the actions tab

**Reply by Marekkon5** [reply]
You click on the "Actions" tab on GitHub, then click on the top most build (it says `Bandcamp match artist` now), then at bottom under the title Artifact you have to download one for your platform (for example `onetagger-win-setup` is for Windows).

**Reply by deedot** [reply]
perfect. I found it as you were responding. ill let you know how it works. thanks again

**Reply by deedot** [reply]
seems to be working. Just cant tell if its really slow or if thats just because its not skipping thousands of tracks anymore

**Reply by Marekkon5** [reply]
You can check the log.


However my guess is that the error was caused by Shazam. There were some issues recently and they added rate limiting, so that might explain it.
